Documentation/usage/extending_danger_two.html.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Extending Danger extended
+subtitle: Plugin engineering
+layout: guide_sw
+order: 0
+blurb: Get your plugin tested
+---
+
+### Writing Tests
+
+Danger Swift makes it easy for you to write tests, it includes a second library for importing into your tests. This
+library includes a set of functions to get a set-up version of the Danger runtime in your tests.
+
+Edit your `Package.swift` to add a reference to the two Danger libraries:
+
+```diff
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "DangerNoCopyrights",
+            dependencies: ["Danger"]),
+        .testTarget(
+            name: "DangerNoCopyrightsTests",
+-           dependencies: ["DangerNoCopyrights"]),
++           dependencies: ["DangerNoCopyrights", "Danger", "DangerFixtures"]),
```

Comment:
I think you don't need Danger here as dependency because is already a dep of `DangerNoCopyrights`